transports/websys-webtransport/.cargo/config.toml

```diff
@@ -0,0 +1,4 @@
+# WebTransport is a working draft! Remove when web-sys moves the interface out of unstable.
+[build]
+rustflags = ["--cfg=web_sys_unstable_apis"]
+rustdocflags = ["--cfg=web_sys_unstable_apis"]
```

Comment:
Note that these only applies when running `cargo build/doc` at this crate level.

Reply:
Would it make sense to add this to the docs.rs table in the `Cargo.toml`?